### ISSUE TITLE
Enums not working properly

### ISSUE THREAD

**Issue by malekjaroslav** [prisma/prisma2#385]
**Version**: prisma2@2.0.0-alpha.119

When using enums and querying  it, it throws an error `photon[mapping.plural].findOne(...)[graphqlField.name] is not a function`

```
enum PricingType {
  NET
  SELLING
}

model Pricing {
  id    String      @default(cuid()) @id @unique
  type  PricingType
  adult Float
  child Float
}
```

Also types are not probably generated correctly
<img width="263" alt="Screenshot 2019-08-20 at 21 50 41" src="https://user-images.githubusercontent.com/7753942/63379468-ac038a00-c394-11e9-91ea-a3dbc52a409d.png">

**Comment by CaptainChemist** [comment]
I believe this is an error with @prisma/nexus. It appears that the fix has been merged in to the next version of @prisma/nexus, but I'm not exactly sure how you'd install it using npm. https://github.com/prisma/nexus-prisma/pull/353

**Comment by malekjaroslav** [comment]
@CaptainChemist Yeah, also just found it. But I was unsuccessful installing it. Probably will need to wait for a release.

**Comment by janpio** [comment]
Ping @jasonkuhrt for confirmation.

**Comment by jasonkuhrt** [comment]
@janpio thanks! Yep, moving this one to `nexus-prisma` repo 👍 

@malekjaroslav 👋 a release will be made this week
